config/renovate/renovate.json

```diff
@@ -566,6 +566,12 @@
     "rebaseWhen": "behind-base-branch",
     "branchTopic": "lock-file-maintenance"
   },
+  "git-submodules": {
+    "enabled": true
+  },
+  "pre-commit": {
+    "enabled": true
+  },
```

Comment:
oh, I thought this `enabled` setting was redundant (at least for `dockerfile` manager), so when I removed the schedules, I deleted them as well.